crates/uv-python/src/discovery.rs

```diff
@@ -720,43 +720,25 @@ impl Error {
                 InterpreterError::Encode(_)
                 | InterpreterError::Io(_)
                 | InterpreterError::SpawnFailed { .. } => true,
-                InterpreterError::UnexpectedResponse { path, .. }
-                | InterpreterError::StatusCode { path, .. } => {
-                    debug!(
-                        "Skipping bad interpreter at {} from {source}: {err}",
-                        path.display()
-                    );
```

Comment:
These messages move to `find_python_installation` so we can avoid displaying a duplicate "skipping" log if we're going to throw it as an error.